src/main/java/net/mcreator/generator/LocalizationUtils.java

```diff
@@ -143,6 +156,9 @@ private static void addLocalizationEntry(Generator generator, String key, Map<?,
 				value = prefix + value;
 
 			generator.getWorkspace().setLocalization(key, value);

```

Comment:
This would be more understandable and efficient if moved to an `else` branch of the following `if`.
This would be more understandable and efficient being executed only if the following `if` condition fails.

Reply:
Done!